locales/sv_SE/pcsx2_Main.po

```diff
@@ -899,32 +899,30 @@ msgstr "Om %s"
 
 #: pcsx2/gui/Dialogs/AboutBoxDialog.cpp:54
 msgid "Website"
-msgstr ""
+msgstr "Webställe"
 
 #: pcsx2/gui/Dialogs/AboutBoxDialog.cpp:56
 msgid "Support Forums"
-msgstr ""
+msgstr "Stödforum"
 
 #: pcsx2/gui/Dialogs/AboutBoxDialog.cpp:58
 msgid "GitHub Repository"
-msgstr ""
+msgstr "GitHub's Magasin"
 
 #: pcsx2/gui/Dialogs/AboutBoxDialog.cpp:60
 msgid "License"
-msgstr ""
+msgstr "Licens"
 
 #: pcsx2/gui/Dialogs/AboutBoxDialog.cpp:62
-#, fuzzy
 msgid "PlayStation 2 Emulator:"
-msgstr "PlayStation®2 emulator"
+msgstr "En PlayStation®2 emulator:"
```

Comment:
Why add 'en' if it's not there in English?

Reply:
Hm, well yes it's incorrect. I'll change it and perhaps make an PR about this being changed into "A PlayStation 2 Emulator" in english later.